config/custom-environment-variables.yml

```diff
@@ -0,0 +1,3 @@
+db:
+  url: DATABASE_URL
```

Comment:
Let's use `DB_URL` and `DB_SCHEMA` to make the configuration. Should we also expose default log level as env variable, e.g. `LOG_LEVEL`?